dev-haskell/stm-hamt/stm-hamt-1.2.1-r1.ebuild

```diff
@@ -19,8 +19,8 @@ RDEPEND=">=dev-haskell/deferred-folds-0.9:=[profile?] <dev-haskell/deferred-fold
 	>=dev-haskell/focus-1:=[profile?] <dev-haskell/focus-1.1:=[profile?]
 	>=dev-haskell/hashable-1.4.0.0:=[profile?] <dev-haskell/hashable-2:=[profile?]
 	>=dev-haskell/list-t-1.0.1:=[profile?] <dev-haskell/list-t-1.1:=[profile?]
-	>=dev-haskell/primitive-0.7:=[profile?] <dev-haskell/primitive-0.10:=[profile?]
-	>=dev-haskell/primitive-extras-0.10.2:=[profile?] <dev-haskell/primitive-extras-0.11:=[profile?]
+	>=dev-haskell/primitive-0.7:=[profile?]
+	>=dev-haskell/primitive-extras-0.10.2:=[profile?]
```

Comment:
Another removal of an upper bound with no change to `CABAL_HACKAGE_REVISION` or `CABAL_CHDEPS`.